pom.xml

```diff
@@ -25,12 +25,12 @@
     <dependency>
       <groupId>cheshire</groupId>
       <artifactId>cheshire</artifactId>
-      <version>5.10.0</version>
+      <version>5.8.1</version>
     </dependency>
     <dependency>
       <groupId>ring</groupId>
       <artifactId>ring-core</artifactId>
-      <version>1.8.2</version>
+      <version>1.8.0</version>
```

Comment:
I'm not in agreement with this at all after some more consideration. Being concerned about *patch* releases which, if following semantic versioning guidelines, will not introduce any breaking changes to third parties. Instead it fixes internal issues that only brings improvements to others. If this was a major or minor change then it would be worth further discussion, but not for patches. These are bugfixes over earlier minor versions.